### PR TITLE
Disabling Express Help Module Bug/324

### PR DESCRIPTION
**Pull request by kevincrafts**
A fresh install or updated site should not display any of the additional help links.

### CODE REVIEW

**Comment by jwfuller** [comment]
@kevincrafts Could you also please also make the follow up removal ticket for the next release?

**Comment by cathysnider** [comment]
@kevincrafts The Help Links that used to appear in upper right of admin pages are no longer there. So, good. 

Suggestion: Remove function cu_help_express_help($variables){} from cu_help.module; since none of that is being used any more.

**Comment by jwfuller** [comment]
This PR should also disable and uninstall `cu_help`. Then we can remove all the code in the next release

**Comment by cathysnider** [comment]
About CU Help module: It does stuff that may belong elsewhere than in a Help module:

https://github.com/CuBoulder/express_mono/blob/455a21afeac488e19b9ae5976c5a0fb832b44db6/modules/custom/cu_help/cu_help.module#L299  adds the link "Enable bundles for additional content types" to bottom of pages 'node/add' and 'block/add'.  It also adds a line of text "Taxonomy is the method used to classify, organize and cross-reference content." to 'admin/structure/taxonomy'. 

We will want to move that functionality elsewhere before we disable cu_help_module.

**Comment by kevincrafts** [comment]
While working on this it was discovered that the help modules provided a few more things than just the in-page help links. These included:

1. Removing the default Drupal help link
2. Adding the Web Express help link to the nav bar
3. Additional links on the "Add Content" and "Add Blocks" screens which linked to the enable bundles screen

I had initially attempted to move some of this code to cu_core and while that passed tests, did not seem do be working on existing sites. This is why this code was all consolidated into cu_help.

This PR aims to remove the api and definitions of the in-page help links and move the additional help-related code to a single module.

File: modules/custom/cu_help/cu_help.module [hunk 1]
```diff
@@ -1,298 +1,44 @@
...
-    $variables);
+function cu_help_menu_alter(&$items) {
+  unset($items['admin/help']);
 }
 
 /**
- * Implements hook_express_help_fields().
+ * Implements hook_navbar().
+ *
+ * Add Help link to navbar.
```
Comment (jwfuller) [inline modules/custom/cu_help/cu_help.module]: Please add a comment that indicates that this is intentionally duplicated code and reference the ticket for removing the older code in the next release.

**Pull request merged**